.github/workflows/build.yml

```diff
@@ -11,7 +11,7 @@ jobs:
   install-go-modules:
     strategy:
       matrix:
-        go: ['1.18.x', '1.17.x', '1.16.x', '1.15.x']
+        go: ['1.19.x']
```

Comment:
Does this constitute a breaking change if we have to restrict  to only 1.19+ going forward? Does this imply we're dropping all support for <1.19?